src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class Duke {
+    private static final String lines = "____________________________________________________________";
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        System.out.println(lines + "\n" + "Hello! I am Pogu, your personal assistant");
+        System.out.println("What do you wish for me to do?" + "\n" + lines);
+
+        String echo = sc.nextLine();
+        ArrayList<Task> arr = new ArrayList<>();
+
+        while(!echo.equals("bye")){
```

Comment:
```suggestion
        while(!echo.equals("bye")) {
```
Wouldn't it be more readable if you add a space after parentheses?